kalite/inline/api_urls.py

```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import include, patterns, url
+
+urlpatterns = patterns (__package__ + '.api_views',   
+    url(r'^(?P<narrative_id>\w+)', 'narrative_view', name="narrative-view")
```

Comment:
I'm not sure what you decided for the `pk` of narratives, but if you decided to use relative urls then I don't think this regular expression will match them.